client/elife-theme/src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import { css } from 'styled-components'
+
+const extraSmall = css`
+  font-size: 11px;
+  line-height: calc(${props => props.theme.gridUnit} * 2);
+  padding: ${props => props.theme.gridUnit};
+`
+
+const small = css`
+  padding: ${props => props.theme.gridUnit};
+`
+
+const primary = css`
+  border: 0 none;
+`
+
+const reverse = css`
+  background: #262626;
+  color: ${props => props.theme.colorTextReverse};
+  border-color: ${props => props.theme.colorTextReverse};
+`
+
+export default css`
+  line-height: calc(${props => props.theme.gridUnit} * 4);
+  min-width: calc(${props => props.theme.gridUnit} * 16);
+  padding: calc(${props => props.theme.gridUnit} * 2);
+  font-size: 14px;
```

Comment:
Should this be `fontSizeBaseSmall`?

Reply:
No, check my discussion with Nick on invision (click "show resolved"). He mentioned that font sizes on buttons should be hardcoded.